plugins/ltac/tactic_debug.ml

```diff
@@ -12,6 +12,7 @@ open Util
 open Names
 open Pp
 open Tacexpr
+open DebugHook
```

Comment:
DebugInterface maybe is a better name?

Reply:
Maybe, but does it really matter?  If you really, really want it...